### DISCUSSION TITLE
some emails body is cropped in normal view in issue description

### DISCUSSION THREAD

**Discussion by shiyamk** [eventum/eventum#1283] [Q&A] [answered]
Hi,

I noticed some email body has been cropped in the normal view but when i switch to plain text it displays all information.

**Normal view**
![image](https://user-images.githubusercontent.com/13827793/144308097-b205241b-da11-4be5-80a0-e587ca274d07.png)

**Plain Text**
![image](https://user-images.githubusercontent.com/13827793/144308569-0d1a0dff-7651-498e-9bc7-d5790d452612.png)

can someone advise me how to fix this or how to set the text format to plain text by default please?

**Comment by vladsf** [comment]
AFAIK there is not way to disable Markdown by default. This must be a bug in markdown previewer (or HTMLPurifier) -  
```
line1: one
line2: two -
here we break: at 12:50:55
```

**Comment by vladsf** [comment, marked as answer]
I found the bug. It is `giberti/commonmark-emoji-extension` breaks the Markdown parser.

Find ` $environment->addExtension(new EmojiExtension());` line in `src/ServiceProvider/MarkdownServiceProvider.php` file and comment
it out with. This hack shall help untile fix is found.

the bug reported - https://github.com/giberti/commonmark-emoji-extension/issues/3

**Reply by shiyamk** [reply]
Thank you. commenting that did work.
